dso-l1/src/main/java/com/tc/object/DistributedObjectClient.java

```diff
@@ -399,6 +402,10 @@ public boolean waitForConnection(long timeout, TimeUnit units) throws Interrupte
     return connectionMade.isSet();
   }
 
+  public Exception getLastDigestedException(){
```

Comment:
Change to something that returns results for attempts to each target destination.